tests/integrations/raftstore/mod.rs

```diff
@@ -6,6 +6,7 @@ mod test_compact_after_delete;
 mod test_compact_lock_cf;
 mod test_compact_log;
 mod test_conf_change;
+mod test_follower_read;
```

Comment:
Prefer `test_replica_read`. Previous tests for replica read is placed in `storage/test_raftkv`, I think it's better to move them here.

Reply:
Let's move them in an individual PR.